src/blocks.zig

```diff
@@ -99,8 +99,11 @@ pub fn register(_: []const u8, id: []const u8, zon: ZonElement) u16 {
 	_mode[size] = rotation.getByID(zon.get([]const u8, "rotation", "cubyz:no_rotation"));
 	_blockHealth[size] = zon.get(f32, "blockHealth", 1);
 	_blockResistance[size] = zon.get(f32, "blockResistance", 0);
+	const rotation_tags = _mode[size].getBlockTags();
+	const block_tags = Tag.loadTagsFromZon(main.worldArena, zon.getChild("tags"));
```

Comment:
For temporary data please use the stackAllocator, we shouldn't needlessly leak stuff into the arena.